client/state/data-layer/wpcom/sites/automated-transfer/initiate/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { initiateTransfer, requestEligibility } from 'state/automated-transfer/actions';
+import { transferStates } from 'state/automated-transfer/constants';
+import {
+	AUTOMATED_TRANSFER_INITIATE_REQUEST,
+} from 'state/action-types';
+
+/**
+ * Maps from API response to internal representation of automated transfer initialize request
+ *
+ * @param {Object} data API response data
+ * @returns {Object} Calypso eligibility information
+ */
+const fromApi = data => Object.assign(
+	{
+		id: data.transfer_id,
+	},
+	data.status === 'active' && { status: transferStates.START },
```

Comment:
> the API endpoint was written before we even started on the calypso side

this is true, but there are two separate endpoints in question here and they provide conflicting names

> It was a very hectic time though

and I believe this haste is the core of why things have been confusing and (for me at least) frustrating